API_Kraken/get_bitcoin_price.py

```diff
@@ -1,20 +1,37 @@
 import requests
 import datetime
 import click
+import time
 
 @click.command()
 @click.option('--data_file', help = 'Path to datafile.')
-def get_price(data_file):
+@click.option('--break_time', default = 5, help = 'Time of a break between prices.')
+
+def get_price(break_time):
```

Comment:
this should be our main method. Please rename it as main for example and create get_price method which only returns new price